script/Deployer.sol

```diff
@@ -123,7 +109,5 @@ contract Deployer is Script {
         escrow.deny(deployer);
         gateway.deny(deployer);
         aggregator.deny(deployer);
-        pauseAdmin.deny(deployer);
-        delayedAdmin.deny(deployer);
```

Comment:
guardian has to deny the deployer as well

Reply:
Guardian.sol doesn't inherit auth, so no need to deny here